drivers/storage/portworx/util/util.go

```diff
@@ -326,6 +330,22 @@ func IsHostPidEnabled(cluster *corev1.StorageCluster) bool {
 	return err == nil && enabled
 }
 
+// IsAutoTLS returns true if the AutoTLS-annotation is true value, and cluster version qualifies
+func IsAutoTLS(cluster *corev1.StorageCluster) bool {
+	enabled, err := strconv.ParseBool(cluster.Annotations[AnnotationIsAutoTLS])
+	if err != nil || !enabled {
+		logrus.WithError(err).Tracef("> IsAutoTLS() early return FALSE")
+		return false
+	}
+
+	if cluster.Spec.Version == "latest" {
```

Comment:
I don't think we have a version set as latest. you can use `pxutil.GetPortworxVersion(cluster)`. if it's a staging tag, it should return a super large version